test/fixtures/server.js

```diff
@@ -67,6 +70,49 @@ function createRawEchoServer () {
     return server;
 }
 
+/**
+ * Simple SSL server for tests. Send
+ *  - 'authorized' to authorized clients.
+ *  - 'unauthorized' to unauthorized clients.
+ *
+ * @param {Boolean} [requestCert=true] - If true, request a certificate from clients that connect
+ *
+ * @example
+ * var s = createSslServer();
+ * s.listen(3000, 'localhost');
+ */
+function createSslServer (requestCert) {
```

Comment:
Accept `key`, `cert` and `ca` from argument just like `requestCert`.
Also, this can be extended to multiple options in the future so it's better to accept the entire options as an argument and still have defaults as a fallback.
Refer: https://github.com/postmanlabs/postman-request/blob/master/tests/server.js#L47